src/FLARM/List.cpp

```diff
@@ -40,3 +40,16 @@ TrafficList::FindMaximumAlert() const noexcept
 
   return alert;
 }
+
+const FlarmTraffic *
+TrafficList::InCloseRange() const noexcept
+{
+  const FlarmTraffic *inrange = NULL;
+
+  for (const auto &traffic : list) {
+    if ((int)traffic.distance < 4000)
+      inrange = &traffic;
+  }
+
+  return inrange;
```

Comment:
Why does this return a pointer? Your API docs suggest it should be a `bool`, and its return value is used as a boolean. Imagine how much simpler this function body could be:
```
  return std::any_of(list.begin(), list.end(), [](const auto &traffic){ return traffic.distance < 4000; });
```

Reply:
Applied as you suggested. Needed a cast for RoughDistance.